client/settings/express-checkout-settings/general-payment-request-button-settings.js

```diff
@@ -259,6 +260,7 @@ const GeneralPaymentRequestButtonSettings = ( { type } ) => {
 					min={ 0 }
 					withInputField={ false }
 					onChange={ setRadius }
+					__nextHasNoMarginBottom
```

Comment:
`36px default size for wp.components.RangeControl is deprecated since version 6.8 and will be removed in version 7.1.`
```suggestion
					__nextHasNoMarginBottom
					__next40pxDefaultSize
```